hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -574,6 +606,21 @@ default Bytes getFailureResultFor(ResponseCodeEnum status) {
 			return creator.createSuccessfulSyntheticRecord(NO_CUSTOM_FEES, sideEffects, EMPTY_MEMO);
 		}
 
+		@Override
+		public long getMinimumFeeInTinybars(Timestamp consensusTime) {
+			Objects.requireNonNull(mintOp);
+
+			switch (mintOp.type()) {
+				case FUNGIBLE_COMMON:
+					return precompilePricingUtils.getMinimumPriceInTinybars(MINT_FUNGIBLE, consensusTime);
+				case NON_FUNGIBLE_UNIQUE:
+					int nftsMinted = Math.max(mintOp.metadata().size(), 1);
+					return precompilePricingUtils.getMinimumPriceInTinybars(MINT_NFT, consensusTime) * nftsMinted;
+				default:
+					return PrecompilePricingUtils.COST_PROHIBITIVE;
```

Comment:
Would this be an unexpected occasion? Maybe a warning log that some case was unhandled?

Reply:
This is unexpected.  The question is how to break it.